Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -918,6 +918,8 @@ Sollten Sie den Test in der App gelöscht haben, können Sie ihn aus dem Papierk
     <string name="qr_code_scan_body_subtitle">"Tests, Ihre Zertifikate, Check-ins und Nachweise bei Ticketbuchungen"</string>
     <!-- YTXT: instruction text for QR code scanning  -->
     <string name="qr_code_scan_body_subtitle_vertretung_warnen">"Sie können Check-ins scannen, um gelöschte Events wiederherzustellen."</string>
+    <!--XTXT: torch accessibility description -->
+    <string name="qr_code_torch_accessibility_description">Taschenlampe</string>
```

Comment:
Is it sufficient to just add a description what it is (Taschenlampe), but not the status (an (on)/aus (off))?

Reply:
The on/off status is already interpreted by Talkback. We were simply missing the description of what exactly is on/off.

Reply:
Ok, thanks.